infra/terraform/main.tf

```diff
@@ -72,3 +72,17 @@ resource "google_container_node_pool" "primary_nodes" {
     }
   }
 }
+
+resource "google_redis_instance" "memorystore_redis_instance" {
+  project        = var.project_id
+  name           = "k8s-redis"
+  tier           = "BASIC"
+  memory_size_gb = var.memory_size_gb
```

Comment:
no need to make this configurable. Just put in the number here.

Also, let's use 16GB instead to give more room than 1GB.